sycl/doc/GetStartedGuide.md

```diff
@@ -400,14 +404,14 @@ if clang was built with the cmake option `SYCL_BUILD_PI_CUDA=ON`.
 The results are correct!
 ```
 **Note**:
-Currently, when the application has been built with the CUDA target, the CUDA backend
-must be selected at runtime using the `SYCL_BE` environment variable. 
+Currently, when the application has been built with the CUDA target, the CUDA
+backend must be selected at runtime using the `SYCL_BE` environment variable.
 
 ```bash
 SYCL_BE=PI_CUDA ./simple-sycl-app-cuda.exe
 ```
 
-NOTE: SYCL developer can specify SYCL device for execution using device
+NOTE: DPC++/SYCL developer can specify SYCL device for execution using device
```

Comment:
Voilà ! :-)

Reply:
I generally don't like slashes in naming, they add confusion most of the times.
In this particular case I guess I wanted to recognize developers who want to use only core SYCL standard features. Now looking into the sentence I'm thinking that saying just "developer" could look better :) Anyway, this is also fine.